Readme.md

```diff
@@ -2,14 +2,16 @@
 [![Build Status](https://travis-ci.com/0lejk4/Scalest.svg?branch=master)](https://travis-ci.com/0lejk4/Scalest)
 
 ## Description
-Scalest is set of helpful features for Akka. 
-First and the main of them is Admin panel which is aggregator of all other features.
-In future there will be healthchecks, more CMS-like features, swagger and cors stuff and far more. 
-The main thing Scalest makes is easier - faster creation of any type of application.
-For instance such admin panel can be very helpful for developing chatbots, 
-as tool for support team or for developer himself.
+
+Scalest is a set of helpful features for Akka.
+The main one of them is the admin panel which is an aggregator of all other features.
+In future there will be healthchecks, more CMS-like features, swagger, cors stuff and much more.
+The main thing Scalest makes easier is fast creation of any type of application.
+For example the admin panel can be very helpful for developing chatbots,
+as a tool for the support team or for the developer themself.
```

Comment:
> It should be "themselves" and If you use "themselves" developer should probably be plural.

"themself" is relatively new word that act's as a gender neural version of himself/herself. Since it is in the [oxford dictionary](https://en.oxforddictionaries.com/definition/themself) and is more appropriate here, I figured  might as well use it here so that I don't change what the original sentence meant.

However in this context, "for the developers themselves" might actually be better.

Reply:
I think last option is the best one.